### ISSUE TITLE
[Question] Post Migration getting error while opening the edit tables 

### ISSUE THREAD

**Issue by cvlmonica** [m-kovalsky/fabric_cat_tools#25]
I have migrated a import mode dataset to direct lake mode ( which has data in Fabric lakehouse). Within it few calculated tables couldn't  be migrated. But our ETL has taken care of it in lakehouse building the same table name with data so while I am trying to click on edit tables so that I can select that missing migrated calculated tables but I am getting this error:

![image](https://github.com/m-kovalsky/fabric_cat_tools/assets/81950014/243872a0-c5a4-47ea-9541-8064ff2f034c)

I have tried multiple times to access but still same error. Even though the semantic model refreshes successful (which means connectivity to that lakehouse is working fine). Any solution around this!?

**Comment by m-kovalsky** [comment]
Did you try opening the semantic model in Tabular Editor to see if there are any errors in the model? Also make sure that the entity partition name for each table matches the corresponding table in your lakehouse.

**Comment by cvlmonica** [comment]
yes, I did checked it in the tabular editor , as I have  a measure named period_selection which is pointing to a calculated table. And while doing the migration that table is not migrated. 

I think if the calculated tables/or any other object which is not migrated then anything related to it  should not be migrated to prevent such errors! What is the solution for this!?

<img width="953" alt="image (5)" src="https://github.com/m-kovalsky/fabric_cat_tools/assets/81950014/b2340f2a-dd5b-4df1-9766-0a2ecc61aadc">

**Comment by m-kovalsky** [comment]
I can append the migration process with a step to remove any objects with errors. It's quite simple and you can even run this code to do it for your own model. This code is for measures but the same principle can be applied to any object.

```python
with connect_semantic_model(dataset='DatasetName', readonly=False, workspace='WorkspaceName') as tom:
    for m in tom.all_measures():
        if m.ErrorMessage != '':
            tom.remove_object(object=m)
```

**Comment by cvlmonica** [comment]
thank you so much!

**Comment by cvlmonica** [comment]
I hope you will be adding a step in migration where if the objects has errors those gets removed from the model.
And what I observed is, consider I have an import mode dataset and I have couple of calculated tables/columns and I have moved these logic into ETL logic and moved those calculated tables as tables into my lakehouse. 
When I ran the migration logic and checks for the validation function it shows that a calculated column is not migrated but actually it has been migrated to the direct lake semantic model.
Is this a bug or what is it!? can you help is understand this!?
Attaching the screenshots below to get a clear understanding:
![image](https://github.com/m-kovalsky/fabric_cat_tools/assets/81950014/123c0cb1-962a-48a7-928a-a56d1e4feffc)


![image](https://github.com/m-kovalsky/fabric_cat_tools/assets/81950014/d2e642ca-7034-4756-8aec-fb26a14eafae)
